go-controller/pkg/util/net.go

```diff
@@ -43,12 +44,13 @@ func IPToUint32(egressIP string) uint32 {
 	return binary.BigEndian.Uint32(ip)
 }
 
-// GetNodeLogicalRouterIP returns the IPs (IPv4 and/or IPv6) of the provided node's logical router
+var forceSNAT = regexp.MustCompile(`lb_force_snat_ip="([^"]*)"`)
+
+// GetNodeLogicalRouterIPs returns the IPs (IPv4 and/or IPv6) of the provided node's logical router
 // Expected output from the ovn-nbctl command, which will need to be parsed is:
 // "chassis=939391b7-b4b3-4c3a-b9a9-665103ee13b5 lb_force_snat_ip=100.64.0.1"
```

Comment:
Given that you've removed `--data=bare` and each field now comes wrapped in quotes: could you update the doc block to reflect that? It gives the impression that the regex is incorrect otherwise.

Reply:
Specifically: ` "chassis=939391b7-b4b3-4c3a-b9a9-665103ee13b5 lb_force_snat_ip=100.64.0.1"` -> `chassis="939391b7-b4b3-4c3a-b9a9-665103ee13b5" lb_force_snat_ip="100.64.0.1"`